doc/ale.txt

```diff
@@ -94,6 +94,7 @@ CONTENTS                                                         *ale-contents*
       htmlhint............................|ale-html-htmlhint|
       tidy................................|ale-html-tidy|
       write-good..........................|ale-html-write-good|
+      stylelint...........................ale-html-stylelint
```

Comment:
Looks like you need to write `|ale-html-stylelint|` there, so it links to the right tag.

Reply:
I didn't even realize there was a `|` there because my vim conceals it 🤦‍♂️